src/styles/utils/_media.scss

```diff
@@ -0,0 +1,15 @@
+
+@media (min-width: 600px) {
+  .container {
+    grid-template-columns: repeat(2, 1fr);
+    grid-template-rows: repeat(3, 1fr);
+  }
+}
+
+@media (min-width: 900px) {
+  .container {
+    grid-template-columns: repeat(3, 1fr);
+    grid-template-rows: repeat(2, 1fr);
+  }
+
+}
```

Comment:
Don't move these media queries to separate files. They should be inside `.container` selector

Reply:
done